frontend/pages/_app.tsx

```diff
@@ -31,7 +32,9 @@ function MyApp({ Component, pageProps }: AppProps) {
     <WagmiConfig client={wagmiClient}>
       <RainbowKitProvider chains={chains}>
         <ChakraProvider theme={theme}>
-          <Component {...pageProps} />
+          <Layout>
```

Comment:
I think we should apply layout to individual pages because we don't want it on all pages (that's why the landing is wonky rn)